.travis.yml

```diff
@@ -0,0 +1,31 @@
+matrix:
+  include:
+
+    - language: java
+      install: true
+      os: linux
+      dist: trusty
+      jdk:
+        - oraclejdk8
+        - openjdk11
+      before_script:
+        - cd java/
+      script:
+        - ./gradlew build
+      after_success:
+        - ./gradlew jacocoTestReport coveralls
+      cache:
+        directories:
+          - $HOME/.gradle
+
+    - language: swift
+      os: osx
+      osx_image: xcode11.3
+      env: SWIFT=5.1
+      before_script:
+        - cd swift/
+      script:
+        - swift build && swift build && swift package generate-xcodeproj && sh package.sh && sh test.sh
```

Comment:
this can be changed to `sh package.sh && sh test.sh`? Is there a place to show the console output for this?

Reply:
Discard the second part. I can see the output from the CI logs.

Reply:
It's already running `sh package.sh && sh test.sh` after build and `swift package`. For the output, just click the travis job below: show all checks -> details on the PR one -> The build -> check the Swift one.

Reply:
Correct. But, we don't want the first two expressions. 

`swift build && swift build && swift package generate-xcodeproj && sh package.sh && sh test.sh` -> `sh package.sh && sh test.sh`. 

This is because `swift build` is called from `package.sh` and we don't need to generate a Xcodeproj for running it on CI.

Reply:
Let me try to make the command steps more readable :)